webcompat/static/js/lib/bugform.js

```diff
@@ -155,6 +156,18 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Event handler for image file drop; purpose to detect multiple drops
+  this.handleImageDrop = function() {
+    var imageFilled = false;
+    if ($(".wc-UploadForm-label.is-hidden").length) {
+      imageFilled = true;
+    }
+    if (imageFilled) {
+      // if 2nd drag and drop make wrapper hidden to make remove button available
+      $(".wc-UploadForm-wrapper").addClass("is-hidden");
```

Comment:
I ended up removing the event handler. previously during testing it was working I think. but currently it was causing the 2nd (+) drag and drops to fail. I cannot recall why it was working/needed before. but currently it appears to make the input field unavailable again.